frontend/src/components/ui/dialog.ts

```diff
@@ -50,5 +50,5 @@ export class Dialog extends SlDialog {
         border-top: 1px solid var(--sl-color-neutral-100);
       }
     `,
-  ];
+  ] as any;
```

Comment:
@emma-sg maybe you have a better solution here, after upgrading shoelace typescript complains about the styles type, I tried `declare static styles: CSSResultGroup` following [this Lit guide](https://lit.dev/docs/components/styles/#inheriting-styles-from-a-superclass) but no go. Any ideas?

Reply:
I believe this is because Shoelace is using a different version of Lit from us, so `SlDialog`'s `styles` field is expecting `CSSResult` from @lit/reactive-element@2.0.2 whereas the `css` function we import from `lit` in our codebase returns a `CSSResult` from @lit/reactive-element@1.6.1. For now I think we can safely just coerce it to the newer version of `CSSResult`: 16611d2

Reply:
Oh except that failed to build, hmm.